### DISCUSSION TITLE
how to remove features  which are from chrX and chrY

### DISCUSSION THREAD

**Discussion by xiaowenchenjax** [stuart-lab/signac#977] [Q&A] [answered]
Dear Developer,
My dataset have female and male samples, I would like to filter all the features from chrX and Y to remove the affect of gender after QC and filtering low-quality cells. 
Could you tell me how I do it? what function I can use?
Thanks.

**Comment by timoast** [comment, marked as answer]
The features used in LSI dimension reduction are set by default to the features in `VariableFeatures()` for the object. If you want to exclude certain features from being used in the dimension reduction (and all downstream functions that use the reduced dimension space, eg clustering), you can set a custom list of features as the variable features using the `VariableFeatures()` assignment function, see here: https://mojaveazure.github.io/seurat-object/reference/VariableFeatures.html
